invenio_rdm_records/services/communities/service.py

```diff
@@ -6,24 +6,100 @@
 # it under the terms of the MIT License; see LICENSE file for more details.
 
 """RDM Record Communities Service."""
+from flask import current_app
+from invenio_records_resources.services import Service, ServiceSchemaWrapper
+from invenio_records_resources.services.uow import RecordCommitOp, unit_of_work
+from kombu import Queue
+from werkzeug.local import LocalProxy
 
-from flask_babelex import lazy_gettext as _
-from invenio_drafts_resources.services.records import RecordService
-from invenio_records_resources.services.uow import unit_of_work
+from invenio_rdm_records.services.errors import (
+    ExceedMaxNumberOfCommunities,
+    RecordCommunityMissing,
+)
 
 
-class RecordCommunitiesService(RecordService):
+class RecordCommunitiesService(Service):
     """Record communities service.
 
     The communities service is in charge of managing communities of a given record.
     """
 
+    @property
+    def indexer(self):
```

Comment:
Yes, you are right. I copy pasted this because I thought that is better to just keep it as small as possible as we have 2 RecordService class that we can inherit, 1 that knows only about records and another that knows about records and drafts, we could inherit abot the one that knows about Records but it still feels like we are getting a lot of things that we don't need.